### ISSUE TITLE
[Screen reader accessibility] Autocomplete does not work with VoiceOver on mac most of the time

### ISSUE THREAD

**Issue by adincebic** [microsoft/vscode#247370]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.99.3
Commit: 17baf841131aa23349f217ca7c570c76ee87b957
Date: 2025-04-15T23:18:46.076Z
Electron: 34.3.2
ElectronBuildId: 11161073
Chromium: 132.0.6834.210
Node.js: 20.18.3
V8: 13.2.152.41-electron.0
OS: Darwin arm64 24.3.0

Steps to Reproduce:

1. Turn VoiceOver on by pressing cmd+F5
2. Open a source file, in my case it is MODULE.bazel
3. Start making changes to stuff which should invoke autocomplete

Expected:
When the autocomplete I can move up or down with keyboard through autocomplete items or keep typing.

Actual:
Seems like something trips up VoiceOverFocus I am unable to keep typing when the autocomplete appears.

A quick remedy is to press shift+tab and that refocuses the editor and I can get back to editing.

Also note that it does not always mess up VoiceOver focus but I am failing to reliably reproduce that.

**Comment by adincebic** [comment]
> Sorry about this. It's now fixed in the [Insiders build](https://code.visualstudio.com/insiders).
> 
> /duplicate https://github.com/microsoft/vscode/issues/246253

Thanks.
